src/plone/restapi/tests/test_batching.py

```diff
@@ -172,10 +174,12 @@ def test_contains_correct_batch_of_items(self):
         response = self.api_session.get("/collection?b_start=2&b_size=2")
 
         # Response should contain second batch of items
-        self.assertEqual(
-            [u"/plone/folder/doc-2", u"/plone/folder/doc-3"],
-            result_paths(response.json()),
+        _result_paths = result_paths(response.json())
+        all_in_collection = all(
```

Comment:
@wesleybl nitpicking: I personally prefer to add all_in_collection directly into the assert. When a variable is only used once, directly after the declaration I prefer to put everything in line. This allows devs to read "in-the-flow" and avoids indirections. I guess this is a matter of preference though.

Reply:
I think declaring declaring the variable name here has value on its own, as it's descriptive of what the assertion will do.

Reply:
As said, it is a matter of preference. IMHO variable naming can never be as descriptive as a code one-liner. This "declare-variable-name" best practice IMHO comes from Java-focused projects where code is not really readable and this might help in some rare cases. In Python the code itself is usually very readable. 

I am a big fan of "clean code" and its best practices. Though, when it comes to this I think Python allows you to derive from this and do better. 

Having to think about a proper variable name alone is something that we could argue about for hours (and that is not worth the effort).

```
self.assertTrue(all_in_collection)
```

Does not give me any clue what this line is actually testing. Even if we would call it "all_result_items_are_present_in_results_set" and think about the variable naming for 5 minutes, you would still have to read it upside-down (first the assert, then the variable declaration) to make sense out of it and which clearly makes it harder to read and process for the human brain.

As said before, this is nitpicking and won't prevent this PR from being merged. Though, my personal opinion and preference is very clear in this case. :)

Reply:
@tisto When I wrote the code the first time, I put everything on one line. So I came across the line:

```python
self.assertTrue(all(path.startswith(u"/plone/folder/doc-") for path in _result_paths))
```

So I thought, "What did I mean by that?"

I found it confusing and then I created the variable. For me, who had it all in my head, it seemed to be clearer. But reading your comment from, it may not be so clear to other peoples.

I will accept your suggestion. Thanks!

Reply:
Done.